signer/src/ws_server/mod.rs

```diff
@@ -43,17 +46,48 @@ impl From<ws::Error> for ServerError {
 	}
 }
 
+/// Builder for `WebSockets` server
+pub struct ServerBuilder {
+	handler: Arc<IoHandler>,
+}
+
+impl Default for ServerBuilder {
```

Comment:
Nope, but it's recommended by rust-clippy to have one if you have a no-argument `pub fn new() -> Self`.
I prefer to use `ServerBuilder::new()` then `ServerBuilder::default()`, but like to keep clippy happy as well:)